src/table/utils/styling-utils.js

```diff
@@ -33,11 +43,13 @@ export function getColor(color = {}, defaultColor, theme) {
 export const getAutoFontColor = (backgroundColor) =>
   isDarkColor(backgroundColor) ? STYLING_DEFAULTS.WHITE : STYLING_DEFAULTS.FONT_COLOR;
 
-export const getBaseStyling = (styleObj, theme) => ({
-  color: getColor(styleObj.fontColor, STYLING_DEFAULTS.FONT_COLOR, theme),
-  fontSize: styleObj.fontSize || STYLING_DEFAULTS.FONT_SIZE,
-  padding: styleObj.fontSize ? `${styleObj.fontSize / 2}px ${styleObj.fontSize}px` : STYLING_DEFAULTS.PADDING,
-});
+export const getBaseStyling = (styleObj, theme) => {
+  return {
+    color: getColor(styleObj.fontColor, STYLING_DEFAULTS.FONT_COLOR, theme),
+    fontSize: styleObj.fontSize || STYLING_DEFAULTS.FONT_SIZE,
+    padding: getPadding(styleObj, STYLING_DEFAULTS.PADDING),
+  };
+};
```

Comment:
Why using a `return` when it is an arrow function?

Reply:
Good question. Leave it for now, not sure why it is disabled.

Reply:
My bad. I forgot to delete it before. A new fix PR will come.

Reply:
ok then let's not change it